standards/feature-definition-of-done.md

```diff
@@ -0,0 +1,10 @@
+# Feature Definition of Done
+
+A feature is done when the following criteria are met.
+
+1. The code compiles without errors
+2. The software built successfully in a CI process
+3. All unit tests pass
```

Comment:
3. can be written more clearly as well.
Maybe "All unit tests have been passed"?